test/base/twilltestcase.py

```diff
@@ -2453,8 +2454,13 @@ def wait_for( self, func, **kwd ):
                 slept += sleep_amount
                 sleep_amount *= 2
             else:
+                exceeded = False
                 break
-        assert slept < walltime_exceeded, 'Tool run exceeded reasonable walltime of 24 hours, terminating.'
+
+        if exceeded:
+            message = 'Tool test run exceeded walltime [total %s, max %s], terminating.' % (slept, walltime_exceeded)
+            log.info(message)
+            raise AssertionError(message)
```

Comment:
The message only appears once in the test log - https://jenkins.galaxyproject.org/job/docker-framework/1322/consoleFull